docs/zh/api-reference/image-generation.md

```
@@ -2,231 +2,167 @@
-## 使用自定义图像工具
+## 请求参数
 
-或者，您可以通过在请求中使用 `image_generation` 工具来生成图像。这种方法提供了对图像生成参数的更多控制。
-
-### 基于工具的图像生成
-
-要使用自定义图像工具，请在请求的 `tools` 数组中包含 `image_generation` 工具：
-
-### 示例
+| 参数 | 类型 | 描述 | 默认值 |
+|-----------|------|-------------|---------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)